File: namsor/php/src/ParseNameGeoExample.php
<?php

namespace OSEG\NamsorExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Namsor;

$config = Namsor\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("api_key", "YOUR_API_KEY");

try {
    $response = (new Namsor\Client\Api\PersonalApi(config: $config))->parseNameGeo(
        name_full: "Ricardo Darín",
        country_iso2: "AR",
    );

    print_r($response);
} catch (Namsor\Client\ApiException $e) {
    echo "Exception when calling PersonalApi#parseNameGeo: {$e->getMessage()}";
}

File: namsor/ruby/src/ParseNameGeoExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["X-API-KEY"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::PersonalApi.new.parse_name_geo(
        "Ricardo Darín", # name_full
        "AR", # country_iso2
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling PersonalApi#parse_name_geo: #{e}"
end
